vet.rsc.OpenRSC.Launcher.json

```
@@ -0,0 +1,62 @@
+{
+    "id": "vet.rsc.OpenRSC.Launcher",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "openrsc",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk17"
+    ],
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--env=PATH=/usr/bin:/app/bin:/app/jre/bin"
+    ],
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands" : [ "/usr/lib/sdk/openjdk17/install.sh" ]
+        },
+        {
+            "name": "openrsc",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D openrsc /app/bin/openrsc",
+                "install -D OpenRSC.jar /app/share/OpenRSC.jar",
+                "install -Dm644 vet.rsc.OpenRSC.Launcher.svg /app/share/icons/hicolor/48x48/apps/vet.rsc.OpenRSC.Launcher.svg",
```

```suggestion
                "install -Dm644 vet.rsc.OpenRSC.Launcher.svg /app/share/icons/hicolor/scalable/apps/vet.rsc.OpenRSC.Launcher.svg",
```

svg is definitely not fixed size.